carma_cooperative_perception/config/cp_sdsm_to_detection_list_node.yaml

```diff
@@ -0,0 +1,28 @@
+# Whether to overwrite the default covariance values with those specified below
+overwrite_covariance: true
```

Comment:
Because covariance is not a standard data provided by the sensor or their drivers, it might be beneficial to overwrite them.